### PR TITLE
improve accessibility of the search fields on the Infusion Docs site

### PR DESCRIPTION
**Pull request by Arraxx**
FLUID-6476
improve accessibility of the search fields on the Infusion Docs site.
screenshot:
Screenshot from 2020-03-12 21-10-37

### CODE REVIEW

**Comment by jobara** [comment]
Could you please add the Issue number to the front of the commit log and PR summary. Something like "FLUID-6476: ... " that way we can track back what the work was done for more easily.

File: src/layouts/default.html.handlebars [hunk 1]
```diff
@@ -91,8 +91,8 @@
                 <nav id="tableOfContents" class="infusion-docs-TOC small-12 medium-3 column" tabindex="-1">
                     {{#ifEqual document.category "Search"}}
                     {{else}}
-                        <span class="infusion-docs-category">Search</span>
-                        <p class="docs-search-mini"><input type="text" name="qs"/></p>
+                        <label for="search" class="infusion-docs-category">Search</label>
```
Comment (jobara) [inline src/layouts/default.html.handlebars]: The `for` attribute needs to point at the `id` of the element it is labelling.

**Comment by coderbabu10** [comment]
okay sir, i will take care of it.

**Comment by greatislander** [comment]
Resolved in #190.

**Pull request closed without merging**